pkgs/by-name/gl/glitchtip/package.nix

```diff
@@ -0,0 +1,131 @@
+{
+  lib,
+  python313,
+  fetchFromGitLab,
+  callPackage,
+  stdenv,
+  makeWrapper,
+  nixosTests,
+}:
+
+let
+  python = python313.override {
+    packageOverrides = final: prev: {
+      django = final.django_5;
+      django-extensions = prev.django-extensions.overridePythonAttrs { doCheck = false; };
+    };
+  };
+
+  pythonPackages =
+    with python.pkgs;
+    [
+      aiohttp
+      anonymizeip
+      boto3
+      brotli
+      celery
+      celery-batches
+      dj-stripe
+      django
+      django-allauth
+      django-anymail
+      django-cors-headers
+      django-csp
+      django-environ
+      django-extensions
+      django-import-export
+      django-ipware
+      django-ninja
+      django-organizations
+      django-prometheus
+      django-redis
+      django-rest-mfa
```

Comment:
Can we fetch https://gitlab.com/glitchtip/glitchtip-backend/-/commit/a34cbea6e6b7f283e74af008e88be8b014b38020 to avoid adding that deprecated and archived dependency?